### ISSUE TITLE
Security Hole: Vulnerable to XSS

### ISSUE THREAD

**Issue by FerrazzzZ** [anchorcms/anchor-cms#989]
I've found several places where a hacker can inject javascript code. For example if there is a post with comments enabled. I can easily just past my following encoded javascript to the comment input
&#x3C;script type=&#x22;text/javascript&#x22;&#x3E;
alert(&#x22;BOOM!!!!!&#x22;);
&#x3C;/script&#x3E;

If the admin approves it on the back end, on the front end you will see the pop up. I've found other vulnerabilities within the code but I haven't been able to test them. I shall be posting more issues.

**Comment by FerrazzzZ** [comment]
"&#x3C;script type=&#x22;text/javascript&#x22;&#x3E;
alert(&#x22;BOOM!!!!!&#x22;);
&#x3C;/script&#x3E;"

**Comment by FerrazzzZ** [comment]
By the way I've noticed the script above doesn't have to be encoded in html entities. If you just past it raw it will still work.

**Comment by beli3ver** [comment]
This bug is fixed in Version 1.0

```html
<div class="content">
	&lt;script type=&quot;text/javascript&quot;&gt;
alert(&quot;BOOM!!!!!&quot;);
 &lt;/script&gt;						
</div>
```

No alert where shown on site or admin panel

**Comment by FerrazzzZ** [comment]
Yes indeed. I just looked at the latest version. They are now using htmlspecialchars
